myapp/lib/passport-config.js

```diff
@@ -31,36 +27,32 @@ passport.use(
       passwordField: 'password',
       passReqToCallback: true,
     },
-    (req, email, password, done) => {
-      connection.query(
-        'SELECT * FROM users WHERE email=?',
-        [email],
-        async (error, results) => {
-          if (results.length > 0) {
-            const user = results[0];
-            // hash化されたパスワードが一致するかチェックする
-            if (await bcrypt.compare(password, user.password)) {
-              // ユーザーを正しく取得できた場合
-              req.session.username = user.name;
-              return done(null, user.email);
-            } else {
-              // パスワードが一致しない場合
-              return done(
-                null,
-                false,
-                req.flash('message', 'ユーザー名またはパスワードが違います')
-              );
-            }
-          } else {
-            // 入力情報に誤りがある場合
-            return done(
-              null,
-              false,
-              req.flash('message', 'ユーザー名またはパスワードが違います')
-            );
-          }
+    async (req, email, password, done) => {
+      let user;
+      try {
+        user = await User.findOne({ where: { email: email } });
```

Comment:
オブジェクトはkeyとvalueが一致している場合、以下のように省略できます。
```suggestion
        user = await User.findOne({ where: { email } });
```